controller/invitation_blackbox_test.go

```diff
@@ -293,6 +294,24 @@ func (s *TestInvitationREST) TestAcceptInvitationFailsForNonUUIDCode() {
 	require.NotNil(s.T(), parameters.Get("error"))
 }
 
+func (s *TestInvitationREST) TestRescindInvitation() {
+	g := s.NewTestGraph()
+	team := g.CreateTeam()
+	invitee := g.CreateUser()
+	inv := g.CreateInvitation(team, invitee)
+
+	service, controller := s.SecuredController(s.testIdentity)
+
+	response := test.RescindInviteInvitationOK(s.T(), service.Context, service, controller, inv.Invitation().InvitationID.String())
+
+	require.NotNil(s.T(), response.Header().Get("Location"))
+
+	// The invitation should no longer be there after rescinding
+	_, err := s.Application.InvitationRepository().Load(s.Ctx, inv.Invitation().InvitationID)
+	require.Error(s.T(), err)
+	require.IsType(s.T(), errors.NotFoundError{}, err)
+}
+
```

Comment:
Perhaps also add tests for a bad request (using an invalid invitation ID, plus a non-uuid invitation id) and unauthorized (have the current user try to rescind an invitation for which they have insufficient privileges).